modules/build/src/main/scala/scala/build/preprocessing/DataPreprocessor.scala

```diff
@@ -15,23 +18,36 @@ case object DataPreprocessor extends Preprocessor {
   ): Option[Either[BuildException, Seq[PreprocessedSource]]] =
     input match {
       case file: VirtualData =>
-        val content = new String(file.content, StandardCharsets.UTF_8)
+        val res = either {
+          val content = new String(file.content, StandardCharsets.UTF_8)
+          val (updatedOptions, dirsLocation) = value {
```

Comment:
```suggestion
          val (updatedOptions, directivesLocation) = value {
```

At first glance I thought it was a variable containing directories. Replace it in another piece of code as well.